bin/ci_free_tds_install.sh

```diff
@@ -0,0 +1,11 @@
+if [ ! -d "tmp/freetds-1.00.108" ]; then
+  mkdir ~/downloads
```

Comment:
Can we put the tarball in `tmp` instead of creating a new directory?

Reply:
I modeled this on https://github.com/pulibrary/figgy/blob/master/bin/ci_simple_tiles_install.sh and the kakadu one also does it this way. do we want a ticket to change all of these?